packages/core/EventManager.ts

```diff
@@ -266,6 +266,10 @@ export default class EventManager {
     };
   }
 
+  public async updateCalendarAttendees(event: CalendarEvent, booking: PartialBooking) {
```

Comment:
I wonder if you can make booking: more specific 💭

Reply:
Do you mean by adding all the fields that should be in booking? Would we just use the Booking type from Prisma client?

Reply:
I think he means that we can be more explicit on which booking properties are needed for this function to work properly instead of just `PartialBooking`.

Reply:
@emrysal what would be best practice then? Would each function have it's own interface? So for this one `booking` would have `updateCalendarAttendeesBooking`?